Source/layered-grammar.lisp

```diff
@@ -1,8 +1,108 @@
 (in-package #:eu.turtleware.polyclot.impl)
 
-(define-class <aesthetic> () ())
-(define-class <stat>      () ())
-(define-class <geom>      () ())
-(define-class <mods>      () ())
-(define-class <scale>     () ())
-(define-class <coord>     () ())
+(define-class <aest> ()
+  (;; mapping
+   (aest :initarg :aest)
+   (vars :initarg :vars)
+   ;; slots used for caching
+   (last-df :initform nil)
+   (indexes :initform nil)))
+
+(define-class <stat> ()
+  ((aest :initarg :aest)))
```

Comment:
```  ((aest :initarg :aest :reader stat-aest)))```

Reply:
as discussed in the issue thread (closing)